pkg/controller/endpointslicemirroring/endpointslicemirroring_controller.go

```diff
@@ -228,6 +228,12 @@ func (c *Controller) Run(workers int, stopCh <-chan struct{}) {
 	<-stopCh
 }
 
+// Queue return Controller ratelimit queue
+// only for testing
+func (c *Controller) Queue() workqueue.RateLimitingInterface {
```

Comment:
why do we want to export the queue and check on the queue len on the tests if we can assert on the slices created?

Reply:
this comment still stands